app/src/main/java/com/lehtimaeki/askold/landingscreen/LandingScreenFragment.kt

```diff
@@ -1,176 +1,198 @@
-            it.first.forEach { iconSet ->
-                data.add(IconSetWrapper(iconSet.id, iconSet, null))
-            }
-
-            if (it.second.isNotEmpty()) {
-                data.add(IconSetWrapper(Int.MAX_VALUE, null, "More fun sets"))
-                it.second.forEach { iconSet ->
-                    data.add(IconSetWrapper(iconSet.id, iconSet, null))
-                }
-            }
-            adapter.dataSet = data
-        })
     }
 
+    @Composable
+    fun TitleText(
+        text: String
+    ) {
+        Text(
+            text,
+            color = Color.Gray,
+            fontSize = 22.sp,
+            modifier = Modifier
+                .padding(start = 16.dp, top = 20.dp, bottom = 16.dp),
+            fontWeight = FontWeight.Bold
+        )
+    }
 
-    inner class IconSetAdapter :
-        RecyclerView.Adapter<RecyclerView.ViewHolder>() {
-
-
-        private val VIEW_TYPE_ICON_SET = 0
-        private val VIEW_TYPE_LABEL = 1
-
-
-        var dataSet: List<IconSetWrapper> = listOf()
-            set(value) {
-                field = value
-                notifyDataSetChanged()
-            }
-
-
-        override fun getItemViewType(position: Int): Int {
-            return when {
-                dataSet[position].iconSet != null -> VIEW_TYPE_ICON_SET
-                else -> VIEW_TYPE_LABEL
-            }
-        }
-
-
-        inner class IconSetViewHolder(view: View) : RecyclerView.ViewHolder(view) {
-            val textView: TextView = view.findViewById(R.id.label)
-            val iconView: ImageView = view.findViewById(R.id.image)
-            val cardView: CardView = view.findViewById(R.id.card)
-        }
-
-        inner class TitleViewHolder(view: View) : RecyclerView.ViewHolder(view) {
-            val textView: TextView = view.findViewById(R.id.label)
-        }
-
-        override fun onCreateViewHolder(
-            viewGroup: ViewGroup,
-            viewType: Int
-        ): RecyclerView.ViewHolder {
-
+    @Composable
+    fun ItemNameText(
+        text: String
+    ) {
+        Text(
+            text,
+            color = Color.Gray,
+            fontSize = 20.sp,
+            modifier = Modifier.padding(start = 16.dp),
+            fontWeight = FontWeight.SemiBold
+        )
+    }
 
-            return when (viewType) {
-                VIEW_TYPE_ICON_SET -> IconSetViewHolder(
-                    LayoutInflater.from(viewGroup.context)
-                        .inflate(R.layout.landing_screen_tile, viewGroup, false)
-                )
-                else -> TitleViewHolder(
-                    LayoutInflater.from(viewGroup.context)
-                        .inflate(R.layout.landing_screen_group_title, viewGroup, false)
-                )
-            }
+    @Composable
+    fun ItemImage(
+        imageId: Int?, modifier: Modifier
+    ) {
+        imageId?.let {
+            Image(
+                painterResource(it), contentDescription = "", modifier = modifier
```

Comment:
null should be used if the view is not important for accessibility, if it is related to an action or event, then a translated string should be used.